schema/defs.yaml

```diff
@@ -30,7 +30,7 @@ description: Common schema definitions shared by all themes
       description: Timestamp when the feature was last updated
       type: string
       format: date-time
-      pattern: ^[1-9]\d{3}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(\.\d{3})?(Z|[-+]\d{2}:\d{2})?$
+      pattern: ^([1-9]\d{3})-(0[1-9]|1[0-2])-(0[1-9]|[12]\d|3[01])T([01]\d|2[0-3]):([0-5]\d):([0-5]\d|60)(\.\d{1,3})?(Z|[-+]([01]\d|2[0-3]):[0-5]\d)$
```

Comment:
Cool.

Uber nit is you might want to either use `\d` consistently or use `[0-9]` consistently, but it's all good.

Reply:
`\d` is used consistently and `[0-9]` is not used. It makes the regex somewhat shorter. What you see here is `[1-9]` used for date elements.